### PR TITLE
Changed talk from SHA1 break to Bughanuta for Hanutas

### PR DESCRIPTION
**Pull request by asuivelentine**
Changed tabs to spaces

### CODE REVIEW

File: content/talks/2017-05-11-Bughanuta for Hanutas.md [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
 created_at: 2017-03-20
 speakers:
-    - Kai
+    - Elias
 author: Matthias Beyer
 slides: false
 tags:
-    - sha
-    - hash
-    - security
```
Comment (matthiasbeyer) [inline content/talks/2017-05-11-Bughanuta for Hanutas.md]: Why removing `security` as tag?

File: content/talks/2017-05-11-Bughanuta for Hanutas.md [hunk 2]
```diff
@@ -1,17 +1,16 @@
 ---
-title: SHA1 Break
+title: Bughanuta for Hanutas
```
Comment (matthiasbeyer) [inline content/talks/2017-05-11-Bughanuta for Hanutas.md:2]: Isn't that kinda doubled here? Because bug _hanuta_ for _hanuta_ ?

**Review by matthiasbeyer — CHANGES_REQUESTED** [review]

File: content/talks/2017-05-11-Bughanuta for Hanutas.md (hunk 2 above)
Comment (asuivelentine) [reply content/talks/2017-05-11-Bughanuta for Hanutas.md:2]: Yeah, It is. But it's also not my talk and that is the title provided by the speaker.

**Review by matthiasbeyer — APPROVED** [review]

**Comment by matthiasbeyer** [comment]
I will do the rebuild this evening. Thanks for the patch!

**Pull request merged**